pkg/storage/unified/resource/client.go

```diff
@@ -17,9 +19,13 @@ import (
 	"github.com/grafana/grafana/pkg/apimachinery/identity"
 	"github.com/grafana/grafana/pkg/services/auth"
 	"github.com/grafana/grafana/pkg/services/authn/grpcutils"
+	"github.com/grafana/grafana/pkg/setting"
 	grpcUtils "github.com/grafana/grafana/pkg/storage/unified/resource/grpc"
 )
 
+// TODO(drclau): decide on the audience for the resource store
+const resourceStoreAudience = "resourceStore"
```

Comment:
No strong opinion what it is called -- do any conventions exist yet?  "resource-store" or "unified-storage" make sense to me.

I do think everything in this client need to be moved up one folder -- into the other `client.go`.  In that case we will need to make the audience public so it can be referenced in both packages